packages/wdio-config/src/lib/ConfigParser.js

```diff
@@ -52,10 +52,16 @@ export default class ConfigParser {
 
             this._config = merge(this._config, fileConfig, MERGE_OPTIONS)
 
+            /**
+             * For Sauce Labs RDC we need to determine if the config file has a `testobject_api_key`
+             * If so, we need to provide a boolean to the `detectBackend` to set the correct hostname
+             */
+            const isRDC = this._capabilities.some(capability => 'testobject_api_key' in capability)
```

Comment:
this line brakes multiremote!

Reply:
Created a fix, my bad, I thought that `capabilites` was always an array